src/task/serverTaskTerminal.ts

```diff
@@ -63,11 +65,16 @@ export class ServerTaskTerminal
 		// At this point we can start using the terminal.
 		if (this._executeServer) {
 			this.running = true;
-			this.writeEmitter.fire('Opening Server...\r\n');
+			this.writeEmitter.fire(
+				localize('openingServer', 'Opening Server...\r\n')

```

Comment:
```suggestion
				localize('openingServer', 'Opening Server...') + '\r\n'

```
so that the translators don't mess with your newline